src/pages-and-resources/pages/PageCard.jsx

```diff
@@ -22,52 +24,57 @@ const CoursePageShape = PropTypes.shape({
 
 export { CoursePageShape };
 
-function PageCard({ intl, page }) {
+function PageCard({
+  intl,
+  page,
+}) {
   const { path: pagesAndResourcesPath } = useContext(PagesAndResourcesContext);
-
-  const pageStatusMsgId = page.isEnabled ? 'pageStatus.enabled' : 'pageStatus.disabled';
   const componentClasses = classNames(
-    'd-flex flex-column align-content-stretch',
-    'bg-light-100 p-3 border shadow',
-    { 'border-gray-500': page.isEnabled, 'border-gray-100': !page.isEnabled },
+    'border shadow card',
+    {
+      'border-gray-500': page.isEnabled,
+      'border-gray-100': !page.isEnabled,
+    },
   );
 
   const handleClick = () => {
-    history.push(`${pagesAndResourcesPath}/${page.id}`);
+    history.push(`${pagesAndResourcesPath}${page.id}`);
```

Comment:
it is breaking url of discussions UI.

Reply:
Yeah this was a temporary change that got committed by mistake